src/lib.rs

```diff
@@ -431,6 +430,55 @@ impl Module {
 		})
 	}
 
+	/// Create `Module` from `parity_wasm::elements::Module`.
```

Comment:
These seems to be not relevant. Let's update this to actually reflect the method's intention

Reply:
Ah crap, I copied it from another method